### PR TITLE
Add Swing popup with credentials on basic/digest Authentication request

### PR DESCRIPTION
**Pull request by serhii-kozionov**
This sample demonstrates how to show the Swing popup window on the basic/digest Authentication request where user can enter his credentials.

### CODE REVIEW

**Comment by serhii-kozionov** [comment]
@vladimir-ikryanov PTAL

**Comment by vladimir-ikryanov** [comment]
This example doesn't work. I don't see any dialog when run it.

**Comment by serhii-kozionov** [comment]
@vladimir-ikryanov PTAL

File: network/src/main/java/AuthenticationHandler.java [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+            frame.add(view, BorderLayout.CENTER);
+            frame.setSize(800, 500);
+            frame.setVisible(true);
+        });
+
+        browser.navigation().loadUrl("http://httpbin.org/basic-auth/user/passwd");
+    }
+
+    private static AuthenticateCallback createAuthenticationPopup(BrowserView view) {
+        return (params, tell) -> SwingUtilities.invokeLater(() -> {
```
Comment (vladimir-ikryanov) [inline network/src/main/java/AuthenticationHandler.java]: Please re-design this example, because `view` can be `null` at this moment which looks bad.

**Comment by serhii-kozionov** [comment]
@vladimir-ikryanov PTAL

**Pull request merged**